build/import/UnitTests.targets

```diff
@@ -1,66 +1,35 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
```

Comment:
This file was only whitespace changes to match the rest of the `.props`/`.targets`/`.*proj` files in the repo.